### PR TITLE
Common Voice dataset

### LINKED ISSUE #66: Mozilla Common Voice
Claimed Set -:  
[Mozilla Common Voice](https://commonvoice.mozilla.org/en/datasets) 

About-
An open-source, multi-language dataset of voices that anyone can use to train speech-enabled applications.

### PR DESCRIPTION
**Pull request by CyberFlameGO**
image
Closes #66

### CODE REVIEW

**Comment by CyberFlameGO** [comment]
This is ready now

**Comment by nirbarazida** [comment]
Hi @CyberFlameGO 

Great work! 
Just before I merge the PR, please add the README file to your DagsHub repo as well. 😄

**Comment by CyberFlameGO** [comment]
It's at the top of the README, though I can put it somewhere else if necessary

**Comment by nirbarazida** [comment]
Not a link to the repo but the README in the repo.
Screen Shot 2021-11-09 at 8 07 11

File: CommonVoice/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+**This dataset is uploaded to a [DAGsHub](https://dagshub.com/CyberFlame/CommonVoice) repository**
+
```
Comment (CyberFlameGO) [inline CommonVoice/README.md:2]:
```suggestion

[DAGShub README](https://dagshub.com/CyberFlame/CommonVoice/src/master/README.md)
```

**Comment by CyberFlameGO** [comment]
I did that and added a link too (okay maybe I didn't properly read your comment until now but shh-)

**Pull request merged**